srcpkgs/CBQN/template

```diff
@@ -0,0 +1,50 @@
+# Template file for 'CBQN'
+pkgname=CBQN
+version=0.2.0
+revision=1
+archs="x86_64*"
+_singeli_hash=853ab1a06ae8d8603f228d8e784fa319cc401459
+_replxx_hash=1da4681a8814366ec51e7630b76558e53be0997d
+_bytecode_hash=78ed4102f914eb5fa490d76d4dcd4f8be6e53417
+hostmakedepends="clang libffi-devel"
```

Comment:
`libffi-devel` should be in `makedepends`.

Reply:
Building for x86_64-musl will fail if `libffi-devel` is in `makedepends` and not `hostmakedepends`.

Reply:
You shouldn't cross-build for `-musl`.  Create a `-musl` `masterdir` and build in it: `./xbps-src -m masterdir-musl binary-bootstrap x86_64-musl ; ./xbps-src -m masterdir-musl pkg CBQN`.